.github/workflows/ci.yml

```diff
@@ -0,0 +1,85 @@
+
+name: Continuous Integration
+
+on: push
+
+jobs:
+  build_test:
+    name: Build and Run Jest Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '8'
```

Comment:
Sanity check, is this project stuck at Node 8? or is this a typo?

Reply:
Not a typo - this is what was pinned in the Travis setup so I kept it there.